src/highdicom/ko/content.py

```diff
@@ -172,7 +172,7 @@ def from_sequence(
                 'Item #1 of sequence is not an appropriate SR Content Item '
                 'because it does not have Template Identifier "2010".'
             )
-        instance = ContentSequence.from_sequence(sequence, is_root=True)
+        instance = ContentSequence.from_sequence(sequence, is_root=is_root)
```

Comment:
I would keep this hardcoded. We just keep the `is_root` parameter to satisfy the [Liskov substitution principle](https://en.wikipedia.org/wiki/Liskov_substitution_principle)

Reply:
Fixed in `0b52144`